regression-test/README.md

```diff
@@ -1,39 +1,31 @@
 # Regression testing app
 
-This app is a target against our regression testing workflow. It simply renders each and every one of the Instructure UI component.
+This is a simple Next.js app importing (locally) @instructure/ui. This allows us to test a couple of different things:
 
-The regression testing suite is fairly simple at this point, it will only check a couple of things:
+- With Cypress and Chromatic we can detect visual changes in our components (e.g. the `<Button/>` component got a larger padding)
+- With Cypress and axe-core we can detect a11y issues (e.g. the `<Button/>` component is missing an `aria-label`)
 
-- use the latest snapshot version of InstUI
-- check if components are in a renderable state (simply render them)
+## Development
 
-## Run the regression tests locally
+### To run this app
 
-In order to run the test you have to have it's dependencies installed. (this project is not a workspace of the main instui project)
+1. Run `npm install` and `npm run bootstrap` from the project root.
+2. Then open the regression test folder: `cd regression-test`
+3. Run the dev server with `npm run dev`
+4. The dev server will be accessible at `localhost:3000`
```

Comment:
Please also write here that one can run the test suite with `npm run cypress`. Also it would be nice to mention regression testing in https://instructure.design/#testing-overview , maybe just link this file on Github `master` branch